### ISSUE TITLE
Snapshot result does not fit preview

### ISSUE THREAD

**Issue by KrizCologne** [hjam40/Camera.MAUI#98]
The feed shown in the CameraView is different to the resulting Snapshot file or ImageSource:

![Screenshot_20230919_072221](https://github.com/hjam40/Camera.MAUI/assets/19465781/b93090b2-ea57-45ab-9ccf-dde534d32e1b)

As you can see, the result-image shows more from the top and less from the bottom. I'm playing around with aspect property of the image but the problem might be that the camerafeed just showing a cutout. 

myView:
```

<VerticalStackLayout VerticalOptions="FillAndExpand" HorizontalOptions="FillAndExpand">
        <cv:CameraView x:Name="cameraView" WidthRequest="300" HeightRequest="300" CamerasLoaded="CameraView_CamerasLoaded" AutoSnapShotFormat="PNG"/>
        <Button Text="Smile" Clicked="Button_Clicked"/>
        <Image x:Name="myIMage" WidthRequest="300" HeightRequest="300" Clip="ToBounds"/>
    </VerticalStackLayout>
```

Code behind:
```
    private async void CameraView_CamerasLoaded(object sender, EventArgs e)
    {
        cameraView.Camera = cameraView.Cameras.First();

        if (cameraView.NumCamerasDetected > 0)
        {
     
            MainThread.BeginInvokeOnMainThread(async () =>
            {
                await cameraView.StopCameraAsync();
                await cameraView.StartCameraAsync();
            });
            
        }
    }
```

```
    private async void Button_Clicked(object sender, EventArgs e)
    {        
        myIMage.Source = cameraView.GetSnapShot();
    }
```

**Comment by jsgg25** [comment]
I had the same issue, the capture didn't match the preview. While reviewing the plugin code, I found something that, in my case, fixed the error
![image](https://github.com/hjam40/Camera.MAUI/assets/116839940/7fc7dee3-eeab-4b07-b1d4-ced2a2ad48b1)

In this line, a 0 was being passed for the x and y positions, which represent the starting point of the bitmap. However, if the preview does not match the maximum screen camera size, that offset must be considered.

` bitmap = Bitmap.CreateBitmap(bitmap, (bitmap.Width - Width)/2, (bitmap.Height - Height) / 2, Width, Height);
`

**Comment by KrizCologne** [comment]
Great, that did the job! @jsgg25 Thank you!

**Comment by aleksandervalle** [comment]
This does indeed fix the issue but this change is not done in the repo, and so the issue should still be open.

**Comment by janusw** [comment]
FYI: This is fixed in release [1.4.7](https://www.nuget.org/packages/CameraMaui/1.4.7), built from [my fork](https://github.com/janusw/Camera.Maui).

**Comment by yassirtl** [comment]
How 

> I had the same issue, the capture didn't match the preview. While reviewing the plugin code, I found something that, in my case, fixed the error ![image](https://private-user-images.githubusercontent.com/116839940/280140269-7fc7dee3-eeab-4b07-b1d4-ced2a2ad48b1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA0MDMxODgsIm5iZiI6MTcxMDQwMjg4OCwicGF0aCI6Ii8xMTY4Mzk5NDAvMjgwMTQwMjY5LTdmYzdkZWUzLWVlYWItNGIwNy1iMWQ0LWNlZDJhMmFkNDhiMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMzE0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDMxNFQwNzU0NDhaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kNzcxZmQ2NmMzYjYxYWE0NGJiOWRmMDRhMDVlYmQ5MTVhNWFiZjllOWYyMDJkZDU5NTYxMTkwMjk3YjBmOGMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.4r_7TLYNRCkql_Hy5Vq11xVytI_c56GQDIdk8DKsVn0)
> 
> In this line, a 0 was being passed for the x and y positions, which represent the starting point of the bitmap. However, if the preview does not match the maximum screen camera size, that offset must be considered.
> 
> `bitmap = Bitmap.CreateBitmap(bitmap, (bitmap.Width - Width)/2, (bitmap.Height - Height) / 2, Width, Height);`

How could i use this bitmap function to solve the problem ?
